client/app/controllers/login-error.js

```diff
@@ -6,6 +6,6 @@ export default class LoginErrorController extends Controller {
     *                     about Contact not existing in CRM.
     */
   get contactNotAssigned() {
-    return this.model.errors.some((error) => error.response.code === 'NO_CONTACT_FOUND');
```

Comment:
Why the move away from `error.response`?
The login error page is now reporting [Object object] once more:
![image](https://user-images.githubusercontent.com/3311663/88974251-2f984d00-d26d-11ea-9e30-8c5ec824a515.png)

Reply:
Again, because the `response` property is not part of the JSON:API spec. It doesn't belong in there.

Reply:
Okay, you're right. We should remove the logging of `error.response` in all our templates. I (or one of us) will do the housekeeping in future PRs.

Reply:
I can do it. I just didn't realize there were other places